yesod-test/Yesod/Test.hs

```diff
@@ -693,6 +694,25 @@ get url = request $ do
     setMethod "GET"
     setUrl url
 
+-- | Follow a redirect, if the last response was a redirect.
+-- ==== __Examples__
+-- > get HomeR
+--
+-- > followRedirect
+followRedirect :: Yesod site
+               =>  YesodExample site ()
+followRedirect = do
+  mr <- getResponse
```

Comment:
I think the error messages that @bitemyapp made for `withResponse` are pretty good, what do you think about using that for this function? https://github.com/yesodweb/yesod/pull/1191/files